Ghidra/Processors/Sparc/data/languages/SparcV9.sinc

```diff
@@ -825,51 +835,73 @@ callreloff: reloc  is disp30 [reloc=inst_start+4*disp30;] { export *:$(SIZE) rel
 
 #----------------MULTIPLY 32 bit
 @if SIZE=="8"
```

Comment:
I'm going to collapse the 32/64 bit variants.  Now that you have fixed the implementation, they perform the same operation.  Only difference is the size of the Y register